### PR TITLE
[FIX] quality_control: created new notebook page to avoid activate variants

### PR DESCRIPTION
**Pull request by oihane**
cc @anajuaristi @pedrobaeza 

This fix is to avoid selecting **"Manage Product Variants"**

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Please try to locate it in other place instead on a new page.

**Comment by oihane** [comment]
The pages available are
* General Information
* Sales (only for product that can be sold)
* Inventory (only for products that are not services, and limited by permission)
* Notes

```xml
<page string="General Information" name="general_information">
<page string="Sales" attrs="{'invisible':[('sale_ok','=',False)]}" name="sales">
<page string="Inventory" name="inventory" groups="stock.group_stock_user,product.group_stock_packaging" attrs="{'invisible':[('type', '=', 'service')]}">
<page string="Notes" name="notes">
```

Should I locate it in **General Information**?

**Comment by pedrobaeza** [comment]
Quality control is applicable to services? I don't remember anything about that, so "Inventory" might be a good place.

File: quality_control/views/product_template_view.xml [hunk 1]
```diff
@@ -11,8 +11,9 @@
         <field name="name">product.template.qc</field>
         <field name="model">product.template</field>
         <field name="inherit_id" ref="product.product_template_only_form_view"/>
+        <field name="groups_id" eval="[(4, ref('quality_control.group_quality_control_manager'))]" />
```
Comment (pedrobaeza) [inline quality_control/views/product_template_view.xml]: A user should view also the triggers IMO

Comment (oihane) [reply quality_control/views/product_template_view.xml]: DONE

**Review by pedrobaeza — APPROVED** [review]

**Review by bodedra — APPROVED** [review]

**Comment by pedrobaeza** [comment]
@lreficent anything by your side that you are using this if I don't remember bad?

**Review by LoisRForgeFlow — APPROVED** [review]
Good for me :+1:

**Pull request merged**